src/game/area.c

```diff
@@ -437,8 +438,10 @@ void play_transition_after_delay(s16 transType, s16 time, u8 red, u8 green, u8 b
 void render_game(void) {
     dynos_update_gfx();
     if (gCurrentArea != NULL && !gWarpTransition.pauseRendering) {
-        geo_process_root(gCurrentArea->unk04, D_8032CE74, D_8032CE78, gFBSetColor);
-
+        if (WAPI.has_focus() || !configNoRenderFocusLoss) {
+            geo_process_root(gCurrentArea->unk04, D_8032CE74, D_8032CE78, gFBSetColor);
```

Comment:
This function is not responsible for rendering the game, this is `gfx_run`.
A part of the game engine is updated during `geo_process_root` (like the famous HOLP and some Mario logic too), as well as some hooks like `HOOK_ON_OBJECT_RENDER`.

Reply:
But the HUD and UI should be still visible

Reply:
could disable interpolation and rendering instead, but i don't know why the hud needs to be visible out of focus

Reply:
If you for example want to read the chat or Star count in the meantime on a second screen without to much resources getting wasted

Reply:
But maybe instead of a checkbox we could one of those djui elements with multiple options